app/scripts/controllers/provisionedService.js

```diff
@@ -0,0 +1,190 @@
+'use strict';
+
+/**
+ * @ngdoc function
+ * @name openshiftConsole.controller:ProvisionedServiceController
+ * @description
+ * Controller of the openshiftConsole
+ */
```

Comment:
I know we have these comments in other files, but I'd prefer just to remove them if we don't have any real description here.